bin/mage_core

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bb
```

Comment:
So it seems you're downloading to `bin/bb`, but still using the system one here, right?

Reply:
fixed this too

Reply:
fixed